src/swapd/runtime.rs

```diff
@@ -1170,11 +1211,13 @@ impl Runtime {
                 };
                 // start fee estimation and block height changes
                 self.syncer_state.watch_fee_and_height(endpoints)?;
-                self.peer_service = peerd.clone();
-                self.enquirer = report_to.clone();
-                if let ServiceId::Peer(ref addr) = peerd {
-                    self.maker_peer = Some(addr.clone());
+
```

Comment:
**Personal opinion on code style alert ;p (i.e. feel free to ignore it)**

I believe it is useful to keep each match arm as one block of code (easier to quickly grasp the block and e.g. for code navigation); what I personally do: either I remove the blank lines or if this needs a separation this is generally a good place to add a small comment that describe the sub-block.

Reply:
6e23cf1a5e53efa983e8bf09ec49a57d3fa4ef64